index.js

```diff
@@ -11,13 +11,47 @@ module.exports = {
     'prettier/standard',
   ],
   rules: {
+    // Prefer () => 0 over () => { return 0 }.
+    'arrow-body-style': 'error',
```

Comment:
So are all these extra rules over and above `standard`?

Reply:
Yes. These are things that `tslint:recommended` enforced for TypeScript. I think that anything we could possibly apply to both should apply to both, so we can be as consistent as possible across both languages. Generally if we get rid of it for one language, we should get rid of it for both.

The only exception I made was `@typescript-eslint/no-var-requires`, which is not autofixable and I felt it would cause too many errors in our existing AngularJS projects. I would like to get rid of `require`s at some point though.